docs/shrc.md

```diff
@@ -236,6 +236,7 @@ Caches can be deleted if they contain many stale classes or if the cache is full
 - [`-Xshareclasses:destroyAllSnapshots`](xshareclasses.md#destroyallsnapshots): Removes all cache snapshots from disk that are found by specifying the `cacheDir` suboption.
 - [`-Xshareclasses:destroyAllLayers`](xshareclasses.md#destroyalllayers): Removes all shared cache layers that are specified by the `name` and `cacheDir` suboptions.
 
+:fontawesome-solid-pencil-alt:{: .note aria-hidden="true"} **Note:** Although you can remove persistent caches and snapshot files from the file system directly, you must always use the utilities to remove non-persistent caches correctly from shared memory.
```

Comment:
> Although you can remove persistent caches and snapshot files from the file system directly

Yes, they can do that. But we may not want to mention this in the doc. We always encourage users to use the utilities to remove caches. Probably remove this and just say

"You must always use the utilities to remove non-persistent caches correctly from shared memory."